akka-docs/src/main/paradox/typed/distributed-data.md

```diff
@@ -41,9 +41,9 @@ out-of-date value.
 
 ## Using the Replicator
 
-The @scala[@unidoc[akka.cluster.ddata.typed.scaladsl.Replicator]]@java[@unidoc[akka.cluster.ddata.typed.javadsl.Replicator]] 
+The @apidoc[typed.*\.Replicator$]
```

Comment:
the main documentation for the `Replicator` is on the `class` not the companion, what will be used in the link?

Reply:
In typed, there is only the object, there is no class (so before this link caused a 404 for the scaladsl).

Reply:
ah, I looked at untyped